go.mod

```diff
@@ -94,7 +95,6 @@ require (
 	go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5 // indirect
 	go.uber.org/atomic v1.7.0 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
-	go.uber.org/zap v1.19.1 // indirect
```

Comment:
If this was done due to a go mod update, please add the command you ran to the commit message.